### ISSUE TITLE
ObjectPage: ResizeObserver on navigation back

### ISSUE THREAD

**Issue by nazarkulyk** [SAP/ui5-webcomponents-react#1300]
On rerender till we navigate back and refresh context ObjectPage cause an error on ResizeObserver, because objectPageRef.current change value to null.

```typescript  
useEffect(() => {
    const fillerDivObserver = new ResizeObserver(() => {
      const maxHeight = Math.min(objectPageRef.current.clientHeight, window.innerHeight);
      ....
  }, [totalHeaderHeight, objectPageRef, children]);
```
![image](https://user-images.githubusercontent.com/22763871/107993394-a105d600-6fda-11eb-8862-ab554dc46d68.png)

![image](https://user-images.githubusercontent.com/22763871/107993438-b7139680-6fda-11eb-8857-243be7b60285.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` "^1.0.0-rc.11"
`@ui5/webcomponents-react` "0.13.9"

Tried to create codesandbox smallest reproducible example, but it looks like our code is more complex :( So i cant reproduce it on small example with codesandbox.

**Comment by nazarkulyk** [comment]
Really thanks for so fast response. Any ETA for next release. Currently we have massive problems with navigation for our Object Pages. Maybe its possible to make small minor release only with this fix ?

**Comment by MarcusNotheis** [comment]
Hey @nazarkulyk,
we expect the new release of Ui5 Web Components today, so we plan to release our next version today as well.
Or do you need it on top of the current 0.13.x releases?

**Comment by MarcusNotheis** [comment]
Released in `@ui5/webcomponents-react@0.13.10`

**Comment by nazarkulyk** [comment]
0.13.10 is version we now move on! Thanks!

**Comment by nazarkulyk** [comment]
@MarcusNotheis problem not solved completely:

![image](https://user-images.githubusercontent.com/22763871/108469299-4691a180-7288-11eb-8547-2527a3997dff.png)

Is it possible still to put fix also for 0.13.x versions ?

Thanks

**Comment by MarcusNotheis** [comment]
Hey @nazarkulyk,
thanks for reporting and sorry for the inconvenience.
We'll look into this and will release the fix for the 0.13.x versions as well (but we can only do this on an exceptional basis, we don't have the capacity to provide long term support for old versions as long as we haven't reached the 1.0.0).
